W/WORLD/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WORLD"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mmorise/World.git", "1f8815f7a85e344ea9f98ca6b1383578a161d4e2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd World/
+ls
+mkdir -p ${libdir}
+$CXX -O1 -Wall -shared -fPIC -o ${libdir}/libworld.${dlext} -Isrc src/*.cpp
```

Comment:
Why `-O1`? Usually people want performance and would want to use higher optimisation levels

Reply:
good point, I copied that from the original makefile where it says `CXXFLAGS = -O1 -Wall -fPIC $(DEBUG_FLAG)`. Would you change it to O2 or even O3?

Reply:
Yes, probably 3 is a good one